src/index.js

```diff
@@ -3,37 +3,18 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import * as serviceWorker from "./serviceWorker";
 import "bootstrap/dist/css/bootstrap.min.css";
-import DocumentManager from "./components/DocumentManager/DocumentManager";
-
-if (!containsCollectionParam(document.location.href)) {
-  insertParam("collection", "d7a2add9-14bf-480e-9b97-96685a006431");
-}
+import DocumentManager from "./components/DocumentManager";
+import { getCollectionFromUrl } from "./util/url";
+import ErrorBoundary from "./components/ErrorBoundary";
 
 ReactDOM.render(
-  <DocumentManager
-    collectionId={getCollectionFromUrl(document.location.href)}
-  />,
+  <ErrorBoundary>
```

Comment:
This assures that if an error occurs anywhere in the app, this top-level error boundary will display instead of a white screen. Note that in dev mode (local), the React error screen for devs will display in front of this, so you'll need to close that to see the ErrorBoundary displayed to end users.

Reply:
Added Error Boundary. Did not pull extra functions out into another file since we have cut it down to a single one liner function